### DISCUSSION TITLE
Poroelastic model

### DISCUSSION THREAD

**Discussion by GGGiozzaG** [SPECFEM/specfem3d#1713] [SPECFEM2D General]
Hi!
Is the poroelastic model sensitive to frequency or assumes low-frequency?
If that is the case, anyone worked out a frequency-dependent solution?

Thanks!

**Comment by cmorency1** [comment]
Hi,
Frequency dependence can be included in the poroelastic model - see the manual section 4.4 and the list of parameters specific to the frequency dependence setting needed in the Par_file, namely the attenuation flags & frequency and quality factor parameters. For more details, please refer to Morency and Tromp (2008) section 8.3.
Hope this will help.
Cheers

**Reply by GGGiozzaG** [reply]
Thanks a lot for your reply! I started reading the reference you suggested.
I'm interested on testing SPECFEM performance at borehole scale. There is only one citation linked to this reference (Sidler et al., 2013), but I am interested if there is additional research on this area to check results and best practices when building up the mesh.
Bests!
